vite-plugin/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "@jahia/vite-plugin",
```

Comment:
As this plugin is public, should we add a README with a "how to" section?

Reply:
Yep! I will take care of this during https://github.com/Jahia/javascript-modules/issues/59
It will be easier to unify all READMEs once the repo is in its "final" form